infrastructure/ecs.tf

```diff
@@ -90,7 +90,8 @@ data "aws_iam_policy_document" "this" {
     effect = "Allow"
     actions = [
       "batch:DescribeJobQueues",
-      "batch:SubmitJob"
+      "batch:SubmitJob",
+      "sagemaker:CreateEndpoint",
     ]
     resources = [
       "*",
```

Comment:
Can we separate out the permissions into two statement blocks; that way we can control what resource the ECS task is allowed to create?

Reply:
Good point @MotwaniM . I have addressed it.